README.md

```diff
@@ -1,6 +1,6 @@
 parodos-dev.github.io 
 
-This site is built by hugo static site generater and published using a github action to https://parodos.dev
+This site is built by hugo static site generater and published using a github action to https://rhdho.dev
```

Comment:
@pkliczewski This change requires a change to the acquired domain.

Reply:
@selrahal thank you for the contribution.

We need to change the domain name but after the release. I am not sure whether we need to use product name upstream. let's discuss this with branding team.